arch/x86/boot/compressed/tpm/tpm2_cmds.c

```diff
@@ -84,11 +84,13 @@ int tpm2_extend_pcr(struct tpm *t, u32 pcr,
 	u16 size;
 	int ret = 0;
 
+	b = alloc_tpmbuff(t->intf, TPM_NO_LOCALITY);
+
 	ret = tpm2_alloc_cmd(b, &cmd, TPM_ST_SESSIONS, TPM_CC_PCR_EXTEND);
 	if (ret < 0)
 		goto out;
 
-	cmd.handles = (u32 *)tpmb_put(b, sizeof(u32));
+	cmd.handles = (u32 *)tpmb_put(b, 2*sizeof(u32));
```

Comment:
I think this is left over from the corrections you all made to the previous version which was missing the tpmb_put for the auth handle. That has been corrected so this ends up allocating an extra handle.

Reply:
Done